db/helpers/common.js

```diff
@@ -143,7 +143,6 @@ function prepareToPublish(inst, changedKeys, ignoreAttributes) {
   for (let i = 0; i < changedKeys.length; i++) {
     if (!ignoreSet.has(changedKeys[i])) {
       return {
-        old: inst._previousDataValues,
         new: inst.get(),
```

Comment:
@shriramshankar Do we have tests for /realtime/socketIOEmitter.js? I don't want to break any functionality by a simple change.